pkg/salud/salud.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package salud monitors the storage radius and request response duration of peers
+// and blocklists peers to maintain network salud (health).
+package salud
```

Comment:
I'd consider moving this package under the `topology` package and renaming it to `healthmonitor`.